website/docs/api/plugins/plugin-content-blog.mdx

```diff
@@ -77,7 +77,7 @@ Accepted fields:
 | `feedOptions.title` | `string` | `siteConfig.title` | Title of the feed. |
 | `feedOptions.description` | `string` | <code>\`$\{siteConfig.title} Blog\`</code> | Description of the feed. |
 | `feedOptions.copyright` | `string` | `undefined` | Copyright message. |
-| `feedOptions.xslt` | <code>boolean \| [FeedXSLTOptions](#FeedXSLTOptions)</code> | `undefined` | Copyright message. |
+| `feedOptions.xslt` | <code>boolean \| [FeedXSLTOptions](#FeedXSLTOptions)</code> | `undefined` | Permits to style the blog XML feeds so that browsers render them nicely with XSLT. |
```

Comment:
```suggestion
| `feedOptions.xslt` | <code>boolean \| [FeedXSLTOptions](#FeedXSLTOptions)</code> | `undefined` | Permits to style the blog XML feeds with XSLT so that browsers render them nicely. |
```

Reply:
Thanks Josh for improving this sentence! Just FYI, I copy-pasted it from below in the file, line 136 :wink: in case you want to correct it too.
Cheers